project_organization.md

```diff
@@ -32,15 +32,23 @@ OTSC meetings agenda and minutes are recorded in [this Google document](https://
 
 OpenTracing has an **Industrial Advisory Board** (OTIAB) that comprises engineers with detailed knowledge of tracing technology at companies of different scales and software maturity. The OTSC members engage with the OTIAB on a periodic basis to gather feedback about successes, challenges, and open-source packages which would benefit from greater OpenTracing support.
 
+The current OpenTracing Industrial Advisory Board is as follows (in alphabetical order):
+
+- Brian Hanafee: Wells Fargo
```

Comment:
Maybe add github usernames as https://github.com/orgs/opentracing/teams/otiab as cannot be made public.